### PR TITLE
input view

### PR DESCRIPTION
**Pull request by kimnamheeee**
-input post view
-input get view
-input delete view

-input model - prompt model 연결
-input serializer

### CODE REVIEW

File: input/models.py [hunk 1]
```diff
@@ -1,8 +1,11 @@
 from django.db import models
 from  django.contrib.postgres.fields import ArrayField
```
Comment (eojin03) [inline input/models.py:2]: 띄어쓰기 이상해요...살려주세용

Comment (sehyeon7) [reply input/models.py:2]: letsgo

File: input/views.py [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
 
 # Create your views here.
+class InputListView(APIView):
+    def post(self, request):
+        pass
+    
+    def get(self, request):
+        prompt_id = request.GET.get('prompt')
+        if not prompt_id:
+            return Response({"detail": "missing fileds ['prompt']"}, status=status.HTTP_400_BAD_REQUEST)
```
Comment (sehyeon7) [inline input/views.py:34]: missing fileds -> missing fields

Comment (kimnamheeee) [reply input/views.py:34]: 헉

**Pull request merged**